### ISSUE TITLE
testEnableAuthentication fails in master

### ISSUE THREAD

**Issue by tedyu** [yugabyte/yugabyte-db#7242]
```
Details:
Expected 2 rows, got 1: [Row[java.lang.String::host,org.postgresql.jdbc.PgArray::{all},org.postgresql.jdbc.PgArray::{all},java.lang.String::all,null,java.lang.String::md5]] expected:<2> but was:<1>
```

**Comment by tedyu** [comment]
Probably related to #6784

**Comment by jaki** [comment]
Fixed by commit 8ef3197813f5f12b3f37cfa1cfedc50ef3d3c77a.
